tests/flow/tests_deprecated_commands.py

```diff
@@ -474,6 +474,18 @@ def test_dagrun_common_errors(env):
     check_error_message(env, con, "PERSIST cannot be specified in a read-only DAG",
                         "AI.DAGRUN_RO PERSIST 1 tensor1{1} |> AI.TENSORSET tensor1{1} FLOAT 1 2 VALUES 5 10")
 
+    # ERR insufficient args for tensor get
+    check_error_message(env, con, "wrong number of arguments for 'AI.TENSORGET' command",
+                        "AI.DAGRUN |> AI.TENSORSET a{1} FLOAT 1 |> AI.TENSORGET")
+
+    # ERR too many args for tensor get
+    check_error_message(env, con, "wrong number of arguments for 'AI.TENSORGET' command",
+                        "AI.DAGRUN |> AI.TENSORSET a{1} FLOAT 1 |> AI.TENSORGET a{1} META BLOB extra")
+
+    # ERR insufficient args for tensor set
+    check_error_message(env, con, "wrong number of arguments for 'AI.TENSORSET' command",
+                        "AI.DAGRUN |> AI.TENSORSET a{1} FLOAT 1 |> AI.TENSORSET")
+
```

Comment:
In your first review you commented that we should test the uncovered code in which are verifying the number of args given to AI.TENSORGET / AI.TENSOSET in the parsing functions. Since we are verifying the arity of these commands in `redisai.c`, the path that covers that code was parsing AI.TENSORGET / AI.TENSOSET IN dag (and actually, I found a potential bug thanks to this check :)